contrib/run.sh

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+# set -ex will print commands
+
+#
+# This script runs the Jenkins server inside the Docker container.
+#
+# It copies the configuration and plugins from /opt/openshift/configuration to
```

Comment:
@waseem-h you can remove these openshift related comments if they are not valid anymore